### PR TITLE
Update package.json/changelog for 3.7.1 release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 3.7.1 (2019-07-02)
+
+- Fixed `skyux build` command to ignore SPA's `fixtures` directories. [#136](https://github.com/blackbaud/skyux-sdk-builder/pull/136)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the `skyux build` command to ignore a SPA's `fixtures` directories.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+- Fixed package dependencies to install missing `ajv` package. [#131](https://github.com/blackbaud/skyux-sdk-builder/pull/131)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the package dependencies to install the missing `ajv` package.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+- Fixed invalid import statement generated by `skyux generate component` command. [#135](https://github.com/blackbaud/skyux-sdk-builder/pull/135)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the invalid import statement generated by the `skyux generate component` command.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
+- Fixed package dependencies to install correct version of `@types/jasmine`. [#134](https://github.com/blackbaud/skyux-sdk-builder/pull/134)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the package dependencies to install the correct version of `@types/jasmine`.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,11 @@
...
+- Removed deprecated `no-use-before-declare` TSLint rule. [#130](https://github.com/blackbaud/skyux-sdk-builder/pull/130)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Removed the deprecated `no-use-before-declare` TSLint rule.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Done.

**Pull request merged**